### DISCUSSION TITLE
Konfiguration HMS-400 DPL Nulleinspeisung - Ratlos

### DISCUSSION THREAD

**Discussion by iroc-fabi** [hoylabs/OpenDTU-OnBattery#1191] [Q&A] [answered]
Hallo Community,
seit gestern habe ich auf der DTU Fusion V2 die aktuelle Version 2024.08.18 am laufen.
Ich habe Probleme bei der Konfiguration bzw. der Funktion DPL.
openDTU regelt die Abgabe leider nicht dynamisch. Bleibt sturr bei %.
Auch jetzt wo wieder PV Erzeugung und damit Netzeinspeisung ist, bleibt es bei 25%.
Denke der wird erst abschalten, wenn der Akku leer ist.

Screenst von gestern Abend:

![Dashboard](https://github.com/user-attachments/assets/9648a7b0-1f45-489f-bda9-0ead2f38c6d8)
![Einstellungen limiter](https://github.com/user-attachments/assets/9a553d13-e96b-46ea-868c-892f39b3fad2)

**Comment by spcqike** [comment]
Kannst du einen log Auszug bereitstellen? Dann könnte man eher nachvollziehen warum da was passiert oder auch nicht.

**Reply by iroc-fabi** [reply]
[Log.txt](https://github.com/user-attachments/files/16736742/Log.txt)

**Reply by schlimmchen** [reply]
`15:10:12.791 > [DPL::calcPowerLimit] power meter value: -6885 W, power meter valid: no, inverter output: 40 W, solar power (AC): 0 W`

Hm. Du hast also den SMA Home Manager laut screenshot. Dann habe ich den gut möglich kaputt-refactor't. Den konnte ich nicht selbst testen aus zeitlichen Gründen und ich habs mir dann schön geredet, weil der am wenigsten angefasst wurde, dass das sicherlich noch gehen würde...

Ich schau mir das mal an später heute oder morgen.

**Reply by iroc-fabi** [reply]
Es ist der Energymeter aber von den Daten egal. Die richtige Einspeisung wird ja angezeigt. Es waren wirklich über 6000W.

wenn du mehr Daten brauchst oder Tests, dann bitte melden.

**Comment by schlimmchen** [comment]
Der DPL hat scheinbar die von die konfigurierte Grundlast eingestellt. Das deutet darauf hin, dass die Verbindung zum PowerMeter ein Problem hat. Welchen PowerMeter verwendest du? Bist du zufällig von #1183 betroffen weil dein Shelly mit einem Passwort geschützt ist? Poste mal deine PowerMeter Config (Screenshots).

**Reply by iroc-fabi** [reply]
![image](https://github.com/user-attachments/assets/39bab060-ae52-4969-b71c-54373705684f)

**Comment by schlimmchen** [comment, marked as answer]
Dein Problem sollte mit 2127b0b0 behoben sein. Du kannst [Firmware herunterladen](https://github.com/helgeerbe/OpenDTU-OnBattery/actions/runs/10540975407), die vom development branch kommt und diese Korrektur enthält. Wäre schön, wenn du die testen könntest.

**Reply by iroc-fabi** [reply]
ja perfekt. Es funktioniert. Wow, sogar sehr flott.

**Reply by schlimmchen** [reply]
Vielen Dank für dein schnelles Feedback!
